Modules/Office365DSC/DSCResources/MSFT_SPOSite/MSFT_SPOSite.psm1

```diff
@@ -875,6 +875,22 @@ function Set-SPOSiteConfiguration
             }
 
             $CurrentParameters = $PSBoundParameters
+            if($CurrentParameters.Template -eq "GROUP#0")
```

Comment:
Can we just add a quick comment at the top to explain what we are doing and why we are removing all the params. It will help down the road.

Reply:
Sure added it in there